build.gradle

```diff
@@ -303,26 +306,58 @@ task coverage {
 task checkStyle {
 }
 
+ext {
+    guiTests = 'address/guitests/*'
+    guiUnitTests = 'address/guiunittests/*'
+    unitTests = 'address/unittests/*'
```

Comment:
No, I mean there will not be a unittests pacakge. instead of `address/unitests/events`, we'll have `address/events` (so that test classes are in the same package as the SUT).